helpers/calendar/ratios_and_utilization.py

```diff
@@ -39,20 +39,23 @@ def get_analytics_ratios(self, query, start, end):
                                                         app_events_list)
 
         ratio_object = RoomAnalyticsRatios().map_results_to_ratio_class(
-                            checkins,
-                            cancellations,
-                            calendar_events_list,
-                            app_events_list)
+            checkins=checkins,
+            cancellations=cancellations,
+            calendar_events_list=calendar_events_list,
+            app_events_list=app_events_list)
 
         return ratio_object
 
-    def get_analytics_ratios_per_room(self, query, start, end):
+    def get_analytics_ratios_per_room(self, query, **kwargs):
```

Comment:
You could have just added ` **kwargs` without removing `start` and `end` and get rid of line 54 and 55